src/libraries/System.Utf8String.Experimental/ref/System.Utf8String.Experimental.csproj

```diff
@@ -1,15 +1,36 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <!-- disable warnings about obsolete APIs -->
     <NoWarn>$(NoWarn);0809;0618</NoWarn>
-    <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netstandard2.1;netcoreapp3.0;$(NetCoreAppCurrent)</TargetFrameworks>
     <Nullable>enable</Nullable>
   </PropertyGroup>
   <ItemGroup Condition="'$(IsPrerelease)' != 'false'">
```

Comment:
Should the whole project be disabled when IsPrerelease=true? None of this should be building in release branches.

Reply:
I guess I assumed that's what this line was intending to do.

@GrabYourPitchforks @joperezr @safern - thoughts on how to do this?

Reply:
> Should the whole project be disabled when IsPrerelease=true? None of this should be building in release branches.

Really?  You don't even want to ship previews?

Here are the set of "modes" that folks usually wish to fall into:
1. Build, ship, and stabilize along with the rest of the product.
2. Build, ship, but do not stabilize along with the rest of the product.
3. Build, but do not ship nor stabilize along with the rest of the product.

Let me know which of these fits the best, or if you have different requirements than what is discussed here.

Reply:
I think we agreed that this should be built in release branches as we still want to ship the experimental packages to nuget but they won't be marked as stable... so to me this condition is no longer needed as we control the package versioning through other properties like: https://github.com/dotnet/runtime/blob/73268c790b7ff31c98a292efbc583e25562074d7/Directory.Build.props#L75-L76

cc: @mmitche @danmosemsft

Reply:
I didn't have @ericstj reply when I was responding, so basically what we do now using the properties above is number 2.

> Build, ship, but do not stabilize along with the rest of the product.

Reply:
>  we still want to ship the experimental packages to nuget

We have not been shipping experimental packages to nuget.org. For example, `System.Runtime.Intrinsics.Experimental` was not on nuget.org for 3.0. People who wanted to use the experimental packages had no problem with using them from the daily feeds.

I see that it have changed and we are publishing `System.Runtime.Intrinsics.Experimental` to nuget.org now: https://www.nuget.org/packages/System.Runtime.Intrinsics.Experimental/ . This is bug that should be fixed.

Reply:
> I think we agreed that this should be built in release branches 

In the discussion that I was in, I believe that we agreed on that experimental packages are:
- Not built in release branches
- Not published on nuget.org
- Service bugs are not serviced in them

Reply:
> We have not been shipping experimental packages to nuget.org. For example, System.Runtime.Intrinsics.Experimental was not on nuget.org for 3.0. People who wanted to use the experimental packages had no problem with using them from the daily feeds.

That was a bug that I introduced in 3.0 and fixed in 5.0.

https://github.com/dotnet/corefx/pull/41513

I had a discussion with @danmosemsft @ericstj and @GrabYourPitchforks and we agreed we wanted them in NuGet to raise visibility and usage of the packages, but they would only ship non-stable versions until they're marked as non-experimental. 

If we want to not have them in NuGet, `IsPrerelease` is not the way to go anyway, we would need to tweak the `IsShippingPackage` property to set it to false for these packages.

Reply:
I agree that publishing experimental packages to nuget.org sends the wrong message. A developer can easily get confused about what is supported.

Keeping experimental packages off of nuget.org, and forcing users to opt-in by using a different package feed, makes it even more clear that this package isn't intended for production usage.

Reply:
I recall the agreement being similar to what Jan said: no building in release branches, no servicing. I'm impartial as to whether or not we publish to nuget.org. Our readme already gives the correct acquisition URL for the _install-package_ command.

Reply:
Opened https://github.com/dotnet/runtime/issues/33636

Reply:
Yeah, I discussed this earlier with @eerhardt and it makes sense we should keep them off nuget as it's inevitable they will be perceived as supported.